command/core.py

```diff
@@ -19,17 +20,19 @@ def handle_app_mention(self, event_data):
         message = event_data["event"]["text"]
         user = event_data["event"]["user"]
         channel = event_data["event"]["channel"]
-        s = message.split(' ', 2)
-        if s[0] != "@rocket":
+
+        if not message.startswith('<@U') and not message.startswith('@'):
             return 0
-        else:
-            command_type = s[1]
-            command = command_type + ' ' + s[2]
-            try:
-                self.__commands[command_type].handle(command, user, channel)
-                return 1
-            except KeyError:
-                return -1
+
+        s = message.split(' ', 2)
```

Comment:
Could some of this be factored out? I assume we'll have essentially the same logic for other event types (e.g. DMs vs channel messages).